### ISSUE TITLE
maven pom flexmark-all is too much

### ISSUE THREAD

**Issue by cekvenich** [neuland/jade4j#187]
only core and util are needed. This would make a smaller/safer jar.

**Comment by wiiitek** [comment]
It looks like

```
<dependency>
    <groupId>com.vladsch.flexmark</groupId>
    <artifactId>flexmark</artifactId>
    <version>0.60.2</version>
</dependency>
```
is enough. See also pull request: https://github.com/neuland/jade4j/pull/190

**Comment by codemonstur** [comment]
The code doesn't actually use flexmark much. There is a single class, MarkdownFilter, inside the codebase that uses this dependency and it only gets instantiated by test classes.

If I may be so bold I would suggest moving MarkdownFilter out of the source and mark flexmark with <scope>test</scope>.

**Comment by chbloemer** [comment]
Merged for 1.3.1.
It would be a possibility to create a separate package only with mardownfilter. And we could remove it from "core".
Maybe in pug4j 2.0. Closing this for now.

**Comment by wiiitek** [comment]
Although `MarkdownFilter` is currently only used in tests, that doesn't mean, that it is not needed.

There might be a possibility, that client code would Like to use it by adding it to `JadeConfiguration`.

It is a public class - part of the API - available for those, who want to use this feature.
